lib/plugins/auto-load.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const _ = require('lodash');
+const debug = require('debug')('voxa');
+
+function autoLoad(skill, adapter, config) {
```

Comment:
i think i'd be better if we just had a `config` object with an `adapter` key, that's what we have in all other plugins